snapcraft.yaml

```diff
@@ -159,6 +159,7 @@ parts:
     - util-linux
     - zfsutils-linux
     - iproute2
+    - diffutils
```

Comment:
Can we move this package in the stage-packages of the containerd part so it is not affacted by the "no-patchelf"?